### ISSUE TITLE
Cut and Paste keys for Mac version do not work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bancika/diy-layout-creator#263]
```
What steps will reproduce the problem?
1. Pressing the Command Key and the "C" keys simultaneously will not copy an 
item
2. Pressing the Command Key and the "V" keys simultaneously will not paste an 
item
3. Pressing the Command Key and the "X" keys simultaneously will not cut an item

What is the expected output? What do you see instead?
Nothing happens

What version of the product are you using? On what operating system?
DIY: v3.28.0
Mac: v10.9.3 OS X Mavericks

```

Original issue reported on code.google.com by `twilytez...@gmail.com` on 10 Jun 2014 at 9:27

**Comment by studleylee** [comment]
After Java 5 or so, the clipboard access was deemed a security issue to please try this:
http://www.shodor.org/interactivate-java/resources/InstructionsCopyPaste/

**Comment by studleylee** [comment]
Oddly I ran into this on a win7 machine yesterday after change and palying with Java version for development. I found I could copy/paste only text and vias, but no other components. The fix was to 
uninstall all my java versions and reinstall with the latest. There was a change a few version back about  ,java.policy that only allowed privileged access to the system clipboard. This was a fallout of the java security issues including where a clipboard could be corrupted and system access executed.

 It made sense and I saw it in the ecIipse debug on the "copy" action on only the mentioned items.
vias and text worked fine. So first went through permutations of trying to add http://www.shodor.org/interactivate-java/resources/InstructionsCopyPaste/ to no avail.

It didn't help so finally a did the above cleanup and reinstall of java and all works perfectly now.
Best Regards, Lee the DiyLayout CheerLeader( male )

**Comment by RightBrainAmp** [comment]
Thanks for this post. I just posted a similar issue and I'm going to try the same steps. It will let me work around my current issue 90% of the time.
Do you have any issues also with using the delete button on your keyboard to eliminate a component and did this fix that issue?

**Comment by mrelwood** [comment]
Use "ctrl" instead of "Command". The correct shortcuts are visible in the Edit menu.

**Comment by pompeiisneaks** [comment]
Yeah came here to say what Mrelwood did, this is a problem with how java works on macs, they forcibly still map copy to CTL-C like on windows, instead of Command like on mac.  I use a java webapp at work and have to remember to do CTL instead of CMD.

**Comment by bancika** [comment]
fixed in 3.39
